medusa/backup_cluster.py

```diff
@@ -28,9 +28,10 @@
 from medusa.network.hostname_resolver import HostnameResolver
 
 
-def orchestrate(config, backup_name, seed_target, stagger, enable_md5_checks, mode, temp_dir,
+def orchestrate(config, backup_name_arg, seed_target, stagger, enable_md5_checks, mode, temp_dir,
                 parallel_snapshots, parallel_uploads):
     backup = None
+    backup_name = backup_name_arg or datetime.datetime.now().strftime('%Y%m%d%H')
```

Comment:
I think we should go down to the minute resolution. Folks may want to perform more than one backup an hour (especially when testing).

Reply:
That's also what I did initially but then I found [this](https://github.com/thelastpickle/cassandra-medusa/blob/a00a45593c44b3e4f99adaa1a89475af987c1a9b/medusa/backup_node.py#L161) so I thought it would be good to be consistent. Imo I would prefer to go down to the minute indeed, wdyt?

Reply:
Yes, I think we should go down to the minute in both places. Would you mind changing the backup-node code to use a minute resolution as well as part of this PR? It would be much appreciated!